tests/utils/dockercli/VolumeCreationTestUtil.go

```diff
@@ -26,26 +26,48 @@ import (
 
 var SSH_OPTS []string = []string{strings.Split(os.Getenv("SSH_KEY_OPT"), " ")[0], strings.Split(os.Getenv("SSH_KEY_OPT"), " ")[1], "-q", "-kTax", "-o StrictHostKeyChecking=no"}
 
-// This util method is going to create vsphere docker volume with
+// CreateDefaultVolume - create vsphere docker volume with
 // defaults.
 func CreateDefaultVolume(ip string, name string) ([]byte, error) {
-	fmt.Printf("\ncreating volume [%s] on VM[%s]", name, ip)
+	fmt.Printf("Creating volume [%s] on VM [%s]\n", name, ip)
 	return InvokeCommand(ip, "docker volume create --driver=vsphere --name="+name)
 }
 
-// This helper deletes the created volume as per passed volume name.
+// AttachVolume - attach volume to container on given host
+func AttachVolume(host, volName, containerName string) ([]byte, error) {
+	fmt.Printf("Attaching volume [%s] on VM[%s]\n", volName, host)
+	return InvokeCommand(host, "docker run -d -v "+volName+
```

Comment:
same as above .. `gofmt` runs as part of the build target which auto corrects the formatting.